src/js/view/view.js

```diff
@@ -868,14 +868,21 @@ function View(_api, _model) {
             _this.trigger(FLOAT, { floating: true });
             _model.set('floating', true);
 
-            _resizeOnFloat = true;
+            if (!_isMobile) {
+                addClass(_wrapperElement, 'jw-float-bottom');
+                _resizeOnFloat = true;
 
-            // Resize within MAX_FLOATING_WIDTH×MAX_FLOATING_HEIGHT bounds, never enlarge.
-            const { width, height } = _this.getSafeRegion(false);
-            const ratio = Math.min(1, MAX_FLOATING_WIDTH / width, MAX_FLOATING_HEIGHT / height);
-            _this.resize(width * ratio, height * ratio, true);
+                // Resize within MAX_FLOATING_WIDTH×MAX_FLOATING_HEIGHT bounds, never enlarge.
+                const { width, height } = _this.getSafeRegion(false);
+                const ratio = Math.min(1, MAX_FLOATING_WIDTH / width, MAX_FLOATING_HEIGHT / height);
+                const floatingWidth = Math.min(MAX_FLOATING_WIDTH, width);
+                const floatingHeight = floatingWidth === MAX_FLOATING_WIDTH ? height * ratio : height;
```

Comment:
@DanFerrer I'm confused what this is trying to do - if I have a video of 398x800, the floating player will be 398x800 with this change? That doesn't seem quite right - it used to be 199x400, which seems more appropriate.

Reply:
@vseventer Part of the AC was to keep the dimensions of the player if the players width was less than 400px.

Reply:
@DanFerrer Ah, you're right - think it's an oversight though, as:
- A player sized 400x800 will float 200x400,
- A player sized 398x800 will float 398x800, and take up a lot / almost all of the vertical height.

Just seems strange / unexpected to me.